src/components/utils/popover.ts

```diff
@@ -381,30 +480,92 @@ export default class Popover extends EventsDispatcher<PopoverEvent> {
       }));
     }
 
+    if (item.isActive) {
+      el.classList.add(Popover.CSS.itemActive);
+    }
+
     return el;
   }
 
   /**
    * Item click handler
    *
    * @param itemEl - clicked item
+   * @param event - click event
    */
-  private itemClicked(itemEl: HTMLElement): void {
+  private itemClicked(itemEl: HTMLElement, event: MouseEvent): void {
     const allItems = this.nodes.wrapper.querySelectorAll(`.${Popover.CSS.item}`);
     const itemIndex = Array.from(allItems).indexOf(itemEl);
     const clickedItem = this.items[itemIndex];
 
-    clickedItem.onClick(clickedItem);
+    this.cleanUpConfirmationState();
```

Comment:
looks like we need to pass the item from which we need to clean that state

```ts
this.cleanUpConfirmationStateFromItem(clickedItem);
```

Reply:
no need in parameter, there can only be one item in confirmation state at the same time. Corresponding html element is queried inside the method